SmurfTestingScripts/README.md

```diff
@@ -38,41 +38,64 @@ because they demand human-approve.
 ### Scenarios
 
 #### keys:
-- setup : set of commands, that are run once on the beginning
-- between_runs : set of commands, that are run in between each test scenario
-- cleanup : set of commands, that are run once in the end of tests
+- setup : list of commands, that are run once at the beginning
+- between_runs : list of commands, that are run in between each test scenario
+- cleanup : list of commands, that are run once at the end of all tests
+- default_executable (optional) : path to the tested executable; the --executable argument has priority over this
 - scenarios : set of testing scenarios containing:
-  - name : test name
-  - timeout : time in seconds after which the run is terminated (SIGTERM). 10 seconds after SIGTERM, SIGKILL is sent 
-  - arguments : set of program arguments. Switch without value has to be an empty string!
+  - name : test name; must be unique
+  - timeout (optional) : time in seconds (can be decimal) after which the run is terminated (SIGTERM). 10 seconds after SIGTERM, SIGKILL is sent 
+  - arguments (optional) : list of program arguments for the tested executable
+  - actions (optional) : list of commands, that are run in parallel with the tested executable
```

Comment:
PLease describe more in details how actions looks like, mechanism of handling etc.

Reply:
added more detailed explanation